elisa/src/org/labkey/elisa/ElisaAssayProvider.java

```diff
@@ -208,8 +208,8 @@ public String getName()
 
     protected Pair<Domain,Map<DomainProperty,Object>> createResultDomain(Container c, User user)
     {
-        Domain dataDomain = PropertyService.get().createDomain(c, "urn:lsid:" + XarContext.LSID_AUTHORITY_SUBSTITUTION + ":" + ExpProtocol.ASSAY_DOMAIN_DATA + ".Folder-" + XarContext.CONTAINER_ID_SUBSTITUTION + ":" + ASSAY_NAME_SUBSTITUTION, "Data Fields");
-        dataDomain.setDescription("The user is prompted to enter data values for row of data associated with a run, typically done as uploading a file.  This is part of the second step of the upload process.");
+        Domain dataDomain = PropertyService.get().createDomain(c, getPresubstitutionLsid(ExpProtocol.ASSAY_DOMAIN_DATA), "Data Fields");
+        dataDomain.setDescription("Define the results properties for this assay design. The user is prompted for these properties for individual rows within the imported run, typically done as a file upload.");

```

Comment:
I believe we're trying to call these "fields" in the UI instead of "properties"

Reply:
Done.